backend/src/api/tasks/tasks.controller.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  ParseIntPipe,
+  Patch,
+  Post,
+  StreamableFile,
+  UploadedFile,
+  UseInterceptors,
+} from "@nestjs/common";
+import {
+  ApiCreatedResponse,
+  ApiForbiddenResponse,
+  ApiNotFoundResponse,
+  ApiOkResponse,
+  ApiTags,
+} from "@nestjs/swagger";
+import { FileInterceptor } from "@nestjs/platform-express";
+import { Express } from "express";
+import "multer";
+import {
+  CreateTaskDto,
+  ExistingTaskDto,
+  UpdateTaskDto,
+  DeletedTaskDto,
+  TaskId,
+} from "./dto";
+import { TasksService } from "./tasks.service";
+import { fromQueryResults } from "../helpers";
+
+@Controller("tasks")
+@ApiTags("tasks")
+export class TasksController {
+  constructor(private readonly tasksService: TasksService) {}
+
+  @Post()
+  @ApiCreatedResponse({ type: ExistingTaskDto })
+  @ApiForbiddenResponse()
+  @UseInterceptors(FileInterceptor("file"))
+  async create(
+    @Body() createTaskDto: CreateTaskDto,
+    @UploadedFile() file: Express.Multer.File,
+  ): Promise<ExistingTaskDto> {
+    const task = await this.tasksService.create(
+      createTaskDto,
+      file.mimetype,
+      file.buffer,
+    );
+
+    return ExistingTaskDto.fromQueryResult({
+      ...task,
+      mimeType: file.mimetype,
+    });
```

Comment:
This should work here 🤔 

```suggestion
    return ExistingTaskDto.fromQueryResult(task);
```

Reply:
will be done in CRT-35